debian/encrypt_files.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+set -euo pipefail
+
+PACKAGE_ENCRYPTION_KEY="${PACKAGE_ENCRYPTION_KEY:-}"
+if [ -z "$PACKAGE_ENCRYPTION_KEY" ]; then
+    echo "ERROR: The PACKAGE_ENCRYPTION_KEY environment variable needs to be set"
+    echo "HINT: If trying to build packages locally, just set it to 'abc' or something"
```

Comment:
thanks for the hint, should we remove this before merge?

Reply:
I think it's useful to keep it in. This is only displayed using package building.